### DISCUSSION TITLE
Front-end repository suggestion

### DISCUSSION THREAD

**Discussion by ptrsmk** [community-scripts/ProxmoxVE#4] [Ideas]
The front-ed repository lacks the ability to open an issue. I think it would be good for that to change.

I noticed this because I wanted to call attention to a minor issue that can be seen in this image:
![image](https://github.com/user-attachments/assets/28ebefc4-098c-41f0-8c3c-8373fdd88269)

Of course, instructions would need to make it very clear that that Issue section is solely related to the actual interface and not any of the scripts.

**Comment by MickLesk** [comment]
@BramSuurdje please take a look

**Reply by bramsuurdje** [reply]
There seems to be an issue that, when images don't load it will instead display the script name again. i'll get on it today, but you can open an issue in the other repository in this organization
